Neos.Neos/Resources/Private/Fusion/Backend/Views/Login.fusion

```diff
@@ -173,3 +161,21 @@ prototype(Neos.Neos:Component.Login.Form) < prototype(Neos.Fusion:Component) {
     </Neos.Fusion.Form:Form>
   `
 }
+
+prototype(Neos.Neos:Component.Login.ImageSource) < prototype(Neos.Fusion:Component) {
```

Comment:
Nitpick: move to a separate file ;)

Reply:
i would name it `Neos.Neos:View.Login.ImageSource` and keep it there 🙈 or just merge it as is

there are parts of `Neos.Neos` that are much dirtier - which need our attention ^^

Reply:
I would also be in favor of outsourcing this part to an external component.

Reply:
Done